### PR TITLE
Update our tech stack based on meeting decisions

### PR DESCRIPTION
**Pull request by QuincyLarson**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `master` branch of Chapter.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,10 +29,10 @@ To better communicate and more easily build an API and UI, we've decided on a co
 
 We are planning to use the following tools:
 
-- [Node.js](https://nodejs.org) / [Express](https://expressjs.com) for our backend
+- The [Node.js](https://nodejs.org) based [Next.js](https://nextjs.org) framework
 - [Postgres](https://www.postgresql.org) with [Sequelize ORM](https://sequelize.org)
 - [Elasticsearch](https://www.elastic.co/what-is/elasticsearch)
-- A [React](https://reactjs.org/) frontend using JavaScript (not TypeScript) and CSS (not Sass)
+- A [React](https://reactjs.org/) frontend using TypeScript) and CSS (not Sass)
```
Comment (Zeko369) [inline README.md]: You left a bracket here

File: README.md [hunk 2]
```diff
@@ -29,10 +29,10 @@ To better communicate and more easily build an API and UI, we've decided on a co
 
 We are planning to use the following tools:
 
-- [Node.js](https://nodejs.org) / [Express](https://expressjs.com) for our backend
+- The [Node.js](https://nodejs.org) based [Next.js](https://nextjs.org) framework
```
Comment (Zeko369) [inline README.md]: We're still technically using express, but I don't really know how to word it, maybe put next down with React and leave express here?

File: README.md (hunk 1 above)
Comment (ScottBrenner) [inline README.md]:
```suggestion
- A [React](https://reactjs.org/) frontend using TypeScript and CSS (not Sass)
```

File: README.md (hunk 2 above)
Comment (vaibhavsingh97) [reply README.md]: We can write Node.js with Express in simple terms?

Comment (Zeko369) [reply README.md]: @vaibhavsingh97 what do you think about
```
Node js with express
React js using next.js framework
```

Comment (vaibhavsingh97) [reply README.md]: SGTM 👍

**Comment by ScottBrenner** [comment]
Merge conflict, need to rebase.

**Comment by vkWeb** [comment]
@QuincyLarson Tech stack has already been updated.

**Pull request closed without merging**